src/commands/settings/serverboard/invite.ts

```diff
@@ -26,38 +26,34 @@ export default class Toggle {
     const guild = interaction.guild;
     const member = guild.members.cache.get(interaction.user.id);
 
-    if (!member.permissions.has(PermissionsBitField.Flags.ManageGuild))
-      return await interaction.followUp({ embeds: [errorEmbed("You need **Manage Server** permissions to add an invite.")] });
+    if (!member.permissions.has(PermissionsBitField.Flags.ManageGuild)) return await interaction.followUp({
+      embeds: [errorEmbed("You need **Manage Server** permissions to add an invite.")]
+    });
+
+    const invite = await serverbTable
+      ?.get(`${guild.id}.invite`)
+      .then(invite => String(invite))
+      .catch(() => "");
 
-    const invite = await serverbTable?.get(`${guild.id}.invite`).then(
-      (invite) => String(invite)
-    ).catch(() => "");
-    if (Boolean(invite)) {
+    if (invite) {
       // Delete invite
```

Comment:
Those comments are really scarcely findable these days, so I don't think this one should stay. I think it's kinda obvious what this thing does as well

Reply:
removed